JobsDeviOS/ContentView.swift

```diff
@@ -7,8 +7,9 @@
 
 import SwiftUI
 
-struct ContentView: View {
-    var body: some View {
+internal struct ContentView: View {
```

Comment:
Eu recomendaria remover a assinatura de internal da struct e as outras assinaturas existentes da classe também. Como internal é o modificador de acesso padrão, você não precisa declará-lo explicitamente.
```suggestion
struct ContentView: View {
```

Reply:
Ajustado